index.bs

```diff
@@ -147,11 +145,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+
+  <h3 id="AVIF-general-brand">AVIF General Brand</h3>
+
+Files conformant with the brand-independent restrictions in this document shall include the brand
+<dfn value="" export="" for="AVIF General brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  Files shall also carry a compatible brand to identify the AVIF profile, if any, with which the file complies.
+
+  <h3 id="image-and-image-collection-files">AV1 image and image collection files</h3>
+  The file extension ".avif" and the MIME Content Type "image/avif" should be used for AVIF files that satisfy the following requirements:
+  - The file does not contain tracks, including image sequences, of any kind.
+  - The [=primary image=] is either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+  - The AVIF file conforms to at least one AVIF profile, and the profile is listed in the [=compatible_brands=] field of the [=FileTypeBox=].
+  - The AVIF file specifies 'avif' in the [=major_brand=] field of the [=FileTypeBox=].
+
+  <h3 id="image-sequence-files">AV1 image sequence files</h3>
+  The file extension ".avifs" and the MIME Content Type "image/avif-sequence" should be used for AVIF files that satisfy the following requirements:
+  - The file contains at least one [=AV1 Image Sequence=].
+  - The [=primary image=] is either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+  - The AVIF file conforms to at least one AVIF profile, and the profile is listed in the [=compatible_brands=] field of the [=FileTypeBox=].
```

Comment:
same remark as for the image file. What about image sequences using AV1 but not conforming to any profile defined today in AVIF?

Reply:
That's what I find strange. If we define a file extension, it should be for all AV1 content. Then you look at the filetypebox (or codecs parameter) and see if you find a profile that you support.

Reply:
But the file extension isn't always tied to the codec. For example, if I put H264 in a ISOBMFF file in some way that is incompatible with the MP4 spec, I can't use the .mp4 file extension.
Or as another example, if a HEIF file uses JPEG for the Primary Item, and uses AV1 for another item, and HEVC for another item, I can't use the .heic file extension just because there happens to be some HEVC in the file.